src/Model/GroundWaterFlow/gwf.f90

```diff
@@ -1257,6 +1258,9 @@ subroutine package_create(this, filtyp, ipakid, ipaknum, pakname, mempath, &
     case ('GHB6')
       call ghb_create(packobj, ipakid, ipaknum, inunit, iout, this%name, &
                       pakname, mempath)
+    case ('GHBA6')
```

Comment:
If we decide that this should be a dev option, we can put a call here to the dev option check.

Reply:
In IDM code just before the new loader is created dev_feature() is called.  This is in IdmMf6File.f90.  This will apply to all new packages that will use the loader so all the base stress packages we plan to update.  Does this accomplish the same thing?